Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/ConceptpowerReply.java

```diff
@@ -200,6 +181,9 @@ public void setWordnetId(String wordnetId) {
         @XmlElement(namespace="http://www.digitalhps.org/", type=ConceptpowerType.class)
         protected ConceptpowerType type;
 
+        @XmlElement(required = true, namespace="http://www.digitalhps.org/")
+        protected AlternativeIds alternativeIds;
```

Comment:
have you tried using an ```@XmlElementWrapper``` annotation here and avoid having an extra class just for a list of strings? Just put it right into a list of strings?